Source/WebCore/editing/Editing.cpp

```diff
@@ -73,6 +74,11 @@ static bool isVisiblyAdjacent(const Position&, const Position&);
 
 bool canHaveChildrenForEditing(const Node& node)
 {
+#if ENABLE(MULTI_REPRESENTATION_HEIC)
```

Comment:
I don't think we should modifying this function.
We should instead override `HTMLPictureElement::canContainRangeEndPoint`.

Reply:
It's the first thing I tried, but that causes many other editing issues. Including an inability to modify the selection using the arrow keys, and clicking around the element to modify the selection.